src/modules/fundingManager/bondingCurve/FM_BC_Bancor_Redeeming_VirtualSupply_v1.sol

```diff
@@ -530,6 +532,13 @@ contract FM_BC_Bancor_Redeeming_VirtualSupply_v1 is
                 Module__FM_BC_Bancor_Redeeming_VirtualSupply__CurveInteractionsMustBeClosed(
             );
         }
+        // Check if virtual supply is big enough to ensure compatibility with relative issuance
+        // token decimal and conversion to 18 decimals done in FM_BC_Tools._convertAmountToRequiredDecimal()
+        // so it will not result in a round down 0 value
+        if (_virtualSupply < 10 ** (issuanceTokenDecimals - 18)) {
```

Comment:
There is no guarantee the issuanceTokenDecimals >= 18, the check when setting it only assures it's >= 7. So this will revert if the decimals are less than 18 and there's no guarantee they aren't